.travis.yml

```diff
@@ -4,8 +4,8 @@ compiler:
 before_install:
     # disable rvm, use system Ruby
     - rvm reset
-    - wget https://raw.githubusercontent.com/yast/yast-devtools/SLE-12-GA/travis-tools/travis_setup.sh
-    - sh ./travis_setup.sh -p "rake yast2-devtools yast2 yast2-core-dev libslp-dev doxygen" -g "rspec:3.3.0 yast-rake gettext"
+    - wget https://raw.githubusercontent.com/yast/yast-devtools/master/travis-tools/travis_setup.sh
+    - sh ./travis_setup.sh -p "rake yast2-devtools yast2 yast2-core-dev libslp-dev" -g "rspec:3.3.0 yast-rake gettext simplecov coveralls:0.8.10 rubocop:0.29.1"
```

Comment:
The idea is to start using them, so I guess I should add the rubocop configuration.